Assets/Scripts/Nodes/FindGod.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using System.Collections;
+
+using BehaviorTree;
+
+public class FindGod : LeafNode
+{
+	private Hashtable _data;
+
+	public override void InitSelf( Hashtable data )
+	{
+		_data = data;
+	}
+
+	public override NodeStatus TickSelf()
+	{
+		GodTag targetGod = GameObject.FindObjectOfType<GodTag>();
```

Comment:
The node is fine, you just have to setup the tree so this only get's called once at the beginning (the same way you'd only call `FindObject()` and cache the value).